lib/cash-in/cash-in-low.js

```diff
@@ -11,6 +11,9 @@ const PENDING_INTERVAL_MS = 60 * T.minutes
 const massage = _.flow(_.omit(['direction', 'cryptoNetwork', 'bills', 'blacklisted', 'addressReuse']),
   convertBigNumFields, _.mapKeys(_.snakeCase))
 
+const massageUpdates = _.flow(_.omit(['cryptoAtoms', 'direction', 'cryptoNetwork', 'bills', 'blacklisted', 'addressReuse']),
+  convertBigNumFields, _.mapKeys(_.snakeCase))
+
```

Comment:
- Probably better to create a const array and _.concat what you need to it instead of having two nearly identical ones.
- I think it's easier if you create the massage function and pick the array you need based on isFinalTxStage() inside the function. That way the if would be hidden away in the massage and you won't need to duplicate code